doc/classes/TileSetScenesCollectionSource.xml

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="TileSetScenesCollectionSource" inherits="TileSetSource" version="4.0">
+	<brief_description>
```

Comment:
Remember to add documentation for this class :slightly_smiling_face:

Reply:
Yeah, I'll do a documentation pass once I'm sure the API is in its final form. For now I only synchronize the docs.